controller.go

```diff
@@ -147,7 +147,7 @@ type initializer struct {
 }
 
 // New creates a new instance of network controller.
-func New(cfgOptions ...config.Option) (NetworkController, error) {
+func New(oldRunningContainers map[string]interface{}, cfgOptions ...config.Option) (NetworkController, map[string]interface{}, error) {
```

Comment:
If the caller is not using the list of restored containers (docker side changes) other than checking the list length, then we should probably return a maskable error instead of adding the new return list.